shishito/reporting/junithtml.py

```diff
@@ -196,6 +196,11 @@ def _appendrow(self, result, report):
                         else:
                             log.append(raw(cgi.escape(line)))
                     log.append(html.br())
+
+                if len(report.sections) > 2:
+                    for stdline in report.sections[2]:
+                        log.append(stdline)
+
```

Comment:
This was actually a bad idea and it was removed in follow up commit.

I should have squashed them :(